docs/explanation/measures.md

```diff
@@ -0,0 +1,299 @@
+# Using the measures framework
+
+## Introduction
+
+The measures framework is used to calculate **quotients** (i.e. a numerator divided by a denominator) and to see how these **vary over time** and when broken down by different **groupings**.
+
+The numerators, denominators and groups are all defined using ehrQL queries, just like you would use with a dataset definition. But the way those definitions are used is a bit different.
+
+To explain the concepts involved we'll start with a basic but complete example and walk through each of the elements.
+
+
+## Basic example
+
+Suppose we want to know what proportion of the patients prescribed atorvastatin tablets in a given month were prescribed 80mg tablets, with the results broken down by sex and calculated for each of the first six months of 2022. Here's how we'd do that using the measures framework:
+
+### Code
+
+```python
+from ehrql import INTERVAL, case, create_measures, months, when
+from ehrql.tables.beta.core import medications, patients
+
+# Every measure definitions file must include this line
+measures = create_measures()
+
+# Small codelist for demonstration purposes; the real list would be longer
+atorvastatin_tablets = [
+    "39733211000001101",
+    "39695411000001103",
+    "39733011000001106",
+    "39733111000001107",
+]
+
+atorvastatin_80mg_tablets = [
+    "39733211000001101",
+]
+
+# The use of the special INTERVAL placeholder below is the key part of
+# any measure definition as it allows the definition to be evaluated
+# over a range of different intervals, rather than a fixed pair of dates
+rx_in_interval = medications.where(
+    medications.date.is_during(INTERVAL)
+)
+atorvastatin_rx = rx_in_interval.where(
+    medications.dmd_code.is_in(atorvastatin_tablets)
+)
+atorvastatin_80_rx = rx_in_interval.where(
+    medications.dmd_code.is_in(atorvastatin_80mg_tablets)
+)
+
+has_recorded_sex = patients.sex.is_in(["male", "female"])
+
+measures.define_measure(
+    name="atorva_80",
+    denominator=atorvastatin_rx.exists_for_patient() & has_recorded_sex,
+    numerator=atorvastatin_80_rx.exists_for_patient(),
```

Comment:
Could you reverse the order of `denominator` and `numerator`? Writing the thing on the bottom of the fraction above the thing on the top of the fraction is adding a small, but noticeable amount of cognitive effort. (I appreciate that you explain the denominator before you explain the numerator, but I don't think writing the arguments in explanation order is that important.)

Reply:
Yeah, I wondered about that too and didn't have a clear sense about which was least worst. Happy to swap to the order here.